.prettierignore

```diff
@@ -0,0 +1,3 @@
+*.md
```

Comment:
Tenhle celý soubor bych vyhodil – Terraform tady nepoužíváme a JSON a MD formátovat chci.

Reply:
Ono to neumi formatovat markdoc a frontmatter u .md. Takze by ti to .md soubory jenom rozbijelo.